qa/suites/fs/upgrade/featureful_client/old_client/centos_9.stream.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/supported-container-hosts/centos_9.stream.yaml
```

Comment:
So I don't think this should work because we don't have centos9 rpms for octopus. The only reason these tests continue to work is that we still build centos8 for `main`:

https://shaman.ceph.com/builds/ceph/main/87dbfb632e974cb2c28d17423071d2d833b53b29/

but that won't be the case much longer.

Looking at this code again, `old_client` always upgrades to quincy and not even to `main` so this should continue to work with `centos_8.stream.yaml`.

We need a ticket to update this suite to actually upgrade to `main` and we'll need some way of exercising the client compatibility feature flags (i.e. whether the MDS should evict the "old client" or not). Probably have the MDS conditionally add some fake debug feature bit which will evict the client rather than rely on an explicit bit.

Reply:
OK, yeh so took the opportunity to delve into fs:upgrade with @dparmar18 PR: https://github.com/ceph/ceph/pull/56476, esp. comment https://github.com/ceph/ceph/pull/56476#issuecomment-2060804292 for a sub-suite wise breakup of what's currently done and what makes sense after a release branch is cut (_next_ main).

Reply:
https://tracker.ceph.com/issues/65580 cc @batrick @dparmar18